templates/api-powershell/dbconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "triggers": {
```

Comment:
i was thinking that we might have a singular config file that would contain all of the "repo" specific "configurations".  maybe it's just named something like repository.json and that file has a database property that is the object you have in this.